src/lifecycle/created.js

```diff
@@ -51,6 +63,7 @@ function markAsResolved (elem, resolvedAttribute, unresolvedAttribute) {
 export default function (opts) {
   let created = opts.created;
   let isNative = opts.isNative;
+  let callUpdateOnProperties = createCallUpdateOnProperties(opts);
   let prototype = applyPrototype(opts.prototype);
```

Comment:
it would be good for these function names to also be more descriptive, maybe something like
`let applyPrototype = createApplyPrototype(opts.prototype);` ?